src/components/context-menu/context-menu.jsx

```diff
@@ -13,7 +13,7 @@ const StyledContextMenu = props => (
 const StyledMenuItem = props => (
     <MenuItem
         {...props}
-        attributes={{className: styles.menuItem}}
+        attributes={{className: styles.menuItem, dividerClassName: styles.divider}}
```

Comment:
This line is causing a test to fail, for some reason (see travis).. I'm not really sure what's wrong though, since this (existing) "className" attribute doesn't cause a similar error, and as far as I can tell these are basically identical.

The code I took note of, BTW, is: https://github.com/vkbansal/react-contextmenu/blob/7580dc0d74b8cbd9ff7ff7f816c71f997f03717f/src/MenuItem.js#L55-L59

Would a better approach to this, rather than specifying `<MenuItem divider />`, be to have a new `StyledMenuItemDivider` / `<MenuItemDivider>` component? That component would specify `attributes.dividerClassName` while `StyledMenuItem` would not.

Reply:
@joker314 OK, sounds good - the diff of that does make sense in fixing it (`+ dividerClassName={undefined}`). That said, I'm curious *why* it's undefined, especially considering that `className` is set properly (`"react-context-menu-item"`). Pushed fix anyway, but, thoughts?

Reply:
@towerofnix Not too sure, but you'll probably want to update https://github.com/LLK/scratch-gui/blob/develop/test/unit/components/sprite-selector-item.test.jsx (and then the snapshot too) to make it not undefined.